spec.html

```diff
@@ -15274,6 +15279,12 @@ <h1>Static Semantics: Early Errors</h1>
           It is a Syntax Error if AssignmentTargetType of |LeftHandSideExpression| is not ~simple~.
         </li>
       </ul>
+      <emu-grammar>AssignmentExpression : LeftHandSideExpression ShortCircuitAssignmentOperator AssignmentExpression</emu-grammar>
```

Comment:
Instead of adding a new section here, you can just modify the preceding early error so it lists this production as well. That's the [existing pattern](https://tc39.es/ecma262/#sec-continue-statement-static-semantics-early-errors).

Reply:
Done.